src/View/Helper/FormHelper.php

```diff
@@ -558,6 +559,11 @@ public function secure(array $fields = [], array $secureAttributes = [])
         if (empty($this->request['_Token'])) {
             return null;
         }
+        $debugSecurity = Configure::read('debug');
+        if (isset($secureAttributes['debugSecurity'])) {
+            $debugSecurity = $secureAttributes['debugSecurity'];
+            unset($secureAttributes['debugSecurity']);
+        }
```

Comment:
Now that the component doesn't support debug mode when `Configure::read('debug')` is false, should this stay? Or should the condition be `isset($secureAttributes['debugSecurity']) && $debugSecurity`